pages/solid-events.md

```diff
@@ -25,7 +17,9 @@ Solid Events provide an opportunity to meet others who are working on Solid or a
 | Date | Event | Organiser | Recordings, Slides and Notes |
 |------|-------|-----------|-----------------------------|
 |2022-06-09| [Solid World](https://www.eventbrite.co.uk/e/solid-world-2022-tickets-223005725127) | [Solid Team](https://solidproject.org/team) | Register for the event over on [Eventbrite](https://www.eventbrite.co.uk/e/solid-world-2022-tickets-223005725127). Agenda will be announced shortly on Eventbrite. |
-|2022-05-12| [Solid World](https://www.eventbrite.co.uk/e/solid-world-2022-tickets-223005725127) | [Solid Team](https://solidproject.org/team) | Presentation: <br/> Flanders Data Utility Company Team: Hear first-hand insight into how the Flemish Data Utility Company wants to stimulate citizens’ trust in sharing data, by focusing on responsible and secure data sharing. The Data Utility Company’s Solid Pod platform is designed to establish trusted data ecosystems on this groundbreaking technology. The Data Utility Company’s platform mitigates the costs and risks associated with innovative technology, lowering the threshold for organizations to enter the Solid ecosystem. |
+|2022-05-12| [Solid World](https://www.eventbrite.co.uk/e/solid-world-2022-tickets-223005725127) | [Solid Team](https://solidproject.org/team) | [Solid World Recording](https://vimeo.com/709214424) <br/> Presentation: <br/> Flanders Data Utility Company Team: Hear first-hand insight into how the Flemish Data Utility Company wants to stimulate citizens’ trust in sharing data, by focusing on responsible and secure data sharing. The Data Utility Company’s Solid Pod platform is designed to establish trusted data ecosystems on this groundbreaking technology. The Data Utility Company’s platform mitigates the costs and risks associated with innovative technology, lowering the threshold for organizations to enter the Solid ecosystem. |
+|2022-05-09 | [Solid Minneapolis](https://www.meetup.com/minneapolis-solid-project/)| [Jon Wilson](https://github.com/orgs/SolidOS/people/YetAnotherJonWilson) |
```

Comment:
I'm getting a 404 on the SolidOS link. Also, it'd be good to link to the specific event page so people can see the event's programme.

```suggestion
|2022-05-09 | [Introducing Solid IRL](https://www.meetup.com/minneapolis-solid-project/events/285299203/) | [Solid Minneapolis](https://www.meetup.com/minneapolis-solid-project/) / [Jon Wilson](https://github.com/YetAnotherJonWilson) |
```